pkg/domain/infra/abi/pods.go

```diff
@@ -226,15 +226,14 @@ func (ic *ContainerEngine) PodRm(ctx context.Context, namesOrIds []string, optio
 		reports []*entities.PodRmReport
 	)
 	pods, err := getPodsByContext(options.All, options.Latest, namesOrIds, ic.Libpod)
-	if err != nil {
+	if err != nil && !(options.Ignore && errors.Cause(err) == define.ErrNoSuchPod) {
```

Comment:
I believe we need to move ignore into `getPodsByContext`. If we specify three pods and the first is missing, we still want the other two. Does `podman pod rm bogus existingA existingB` work correctly and removes the last two?

Reply:
I had a look at the code and it's fine. `getPodsByContext` runs over all specified ctrs/pods.